proposals/0315-Add-RPC-Conflict-Management.md

```diff
@@ -22,22 +22,30 @@ The example above shows that the latter RPC `PerformAudioPassThru` is displayed
 To solve these problems, we propose to add a new RPC conflict management function to SDL Core.
 
 ## Proposed solution
-To implement the RPC conflict management, we will add a new RPC conflict management module, `InterruptManager`, and configuration file, `InterruptManagerConfig`, to SDL Core.
+To implement the RPC conflict management, we will add a new RPC conflict management module, `InterruptManager` to SDL Core, and add `InterruptManagerConfig` to policy table.
 
-RPC priority table and `AppHMIType` priority table are specified in `InterruptManagerConfig`. RPC priority table and `AppHMIType` priority table are tables that each set the priority of RPC and app type (`appHMIType`). By modifying `InterruptManagerConfig`, an OEM can receive the expected request from SDL Core during RPC conflict. On the other hand, `InterruptManager` reads `InterruptManagerConfig` during the SDL Core startup and builds the two tables mentioned above based on their settings. When an RPC conflict occurs, the `InterruptManager` first determines the RPC with a high priority according to the RPC priority table. However, if two competing RPCs have the same priority, then the RPC with the higher priority is determined according to the `AppHMIType` priority table.
+RPC priority table and `App Priority` table are specified in `InterruptManagerConfig`. RPC priority table and `App Priority` table are tables that each set the priority of RPC and app. By modifying `InterruptManagerConfig`, an OEM can receive the expected request from SDL Core during RPC conflict. On the other hand, `InterruptManager` reads `InterruptManagerConfig` in policy table during the SDL Core startup and builds the two tables mentioned above based on their settings. When an RPC conflict occurs, the `InterruptManager` first determines the RPC with a high priority according to the RPC priority table. However, if two competing RPCs have the same priority, then the RPC with the higher priority is determined according to the `App Priority` priority table. Therefore, only one modal RPC to be displayed at a time.
```

Comment:
Can you please advise which [agreed upon revision](https://github.com/smartdevicelink/sdl_evolution/issues/1068#issuecomment-704467722) requested changing `AppHMIType` priority table to `App Priority` table?

Reply:
@theresalech -san,
The RPC priority of previous proposal is below.
1. RPC priority
2. appHMIType priority
However, In review, the RPC priority order was chaged as below.
1. Apps with app priority EMERGENCY
2. RPC priority
3. App priority(non-emergency)
4. HMI Status
Since the priority is replaced from appHMIType to app priority, the appHMIType was not used in the priority order. Therefore, I removed the appHMIType section.